contracts/interface/ISlashIndicator.sol

```diff
@@ -2,7 +2,7 @@ pragma solidity 0.6.4;
 
 interface ISlashIndicator {
     function clean() external;
-    function downtimeSlash(address validator, uint256 count) external;
```

Comment:
I will suggest do not include this in the event `shouldRevert`

Reply:
This is not an event. It's the function interface